Dockerfile

```diff
@@ -17,7 +17,7 @@ ENV PATH $PATH:$GOROOT/bin:$GOPATH/bin
 
 RUN go get github.com/openshift/origin && \
     hack/build-go.sh && \
-    cp _output/local/go/bin/* /usr/bin/ && \
+    cp _output/local/bin/linux/amd64/* /usr/bin/ && \
```

Comment:
For Linux today yes, in the future there may be arm architectures.

On Sep 27, 2015, at 11:55 AM, Jordan Liggitt <notifications@github.com>
wrote:

In Dockerfile
<https://github.com/openshift/origin/pull/4816#discussion_r40505922>:

> @@ -17,7 +17,7 @@ ENV PATH $PATH:$GOROOT/bin:$GOPATH/bin
>
>  RUN go get github.com/openshift/origin && \
>      hack/build-go.sh && \
> -    cp _output/local/go/bin/* /usr/bin/ && \
> +    cp _output/local/bin/linux/amd64/* /usr/bin/ && \

Assumes current platform is linux/amd64... Is that valid?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4816/files#r40505922>.